dev-server/ui-playground/components/patterns/SharedButtonPatterns.js

```diff
@@ -15,10 +15,27 @@ export default function SharedButtonPatterns() {
   return (
     <PatternPage title="Buttons (Shared)">
       <Pattern title="IconButton">
-        <p>A button containing an icon and no label (text)</p>
+        <p>A button containing an icon and no other content.</p>
```

Comment:
Might be easier to just look at this in a browser (locally, pattern library area). I don't imagine these changes need to be reviewed with any fine-toothed comb.